protected/humhub/modules/space/models/Membership.php

```diff
@@ -159,35 +165,61 @@ public function countNewItems()
      * @param boolean $cached use cached result if available
      * @return Space[] an array of spaces
      */
-    public static function GetUserSpaces($userId = "", $cached = true)
+    public static function getUserSpaces($userId = '', $cached = true)
```

Comment:
For I know, on Windows `ClassName === className` is true, on Linux it will syntax error, about the name of methods I don't know. Are you sure about getUserSpaces for this case isn't BC ?

Reply:
PHP does not care about method case, the classname issue on windows/linux is about file names and autoloading. No autoloading is involved in this case, so its fine.

>     Note: Function names are case-insensitive, though it is usually good form to call functions as they appear in their declaration. 
> https://secure.php.net/manual/en/functions.user-defined.php